examples/pytorch/llm/llm_infer.py

```diff
@@ -45,25 +45,17 @@ def parse_args() -> Arguments:
     return args
 
 
-args = parse_args()
+COMMAND_LINE_MODE = True
```

Comment:
write following code in a function maybe called inference,  can call it in the following code for command line mode, and in jupyter notebook , user can construct Aguments and pass it to inference by themselves

if __name__ == '__main__':
    args = parse_args()
    inference()